components/server/ee/src/prebuilds/prebuilt-status-maintainer.ts

```diff
@@ -175,7 +175,10 @@ export class PrebuildStatusMaintainer implements Disposable {
                         throw err;
                     }
                 }
-                span.log({ 'update': 'done', 'found': found });
+                TraceContext.addNestedTags(ctx, {
+                    update: 'done',
+                    found
```

Comment:
I think it's worth trying to namespace these. We don't have a convention established for how to generically namespace attributes for custom spans, but how just going namespacing them under the name of the span? 

That would make it `doUpdate.update` and `doUpdate.found`.

I think that gives us fairly readable queries. E.g. to find attributes related to the span you care about you'd type `doUpdate.` and use code completion to explore the attributes. Queries would then textually looks something like:

```
COUNT
WHERE name = doUpgrade
GROUP BY doUpgrade.update, doUpgrade.found
```

Reply:
Good point, will do.